test/test_artifacts/v1/sagemaker-headless-execution-driver.test.Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+# Execute the unit tests for sagemaker-headless-execution-driver
+CMD ["python", "-c",  "import sagemaker_headless_execution_driver.headless_execution as execution_driver"]
```

Comment:
Can you add the same unit test changes mentioned here: https://github.com/aws/sagemaker-distribution/pull/105

Reply:
Discussed offline. Will merge for now and circle back to update this